quotes-react-app/src/Main/Main.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from "react";
+
+export const Main = () => {
+  const [quotes, setQuotes] = useState([]);
+  const [randomQuote, setRandomQuote] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
In general it's great to minimise the number of different state variables we have where we can, because it means there are fewer combinations we need to think about.

Is there a way you can know the page is loading _without_ having a separate `isLoading` state?